portal/server/routes/contract/about/supplierPage.js

```diff
@@ -73,7 +83,6 @@ router.get('/contract/:_id/about/supplier', [validateRole({ role: [MAKER] }), pr
     'indemnifier-address-country': mapCountries(countries, indemnifierAddressCountryCode),
     'indemnifier-correspondence-address-country': mapCountries(countries, indemnifierCorrespondenceAddressCountryCode),
   };
-
```

Comment:
could we reinsert these line breaks to improve readability please

Reply:
Done